packages/coffeescript/plugin/compile-coffeescript.js

```diff
@@ -83,8 +83,8 @@ export class CoffeeCompiler extends CachingCompiler {
 
     let sourceMap = JSON.parse(output.v3SourceMap);
 
-    if (source.match(/(`|import|export|function\*)/)) {
-      // If source contains backticks or ES2015+ features,
+    if (source.match(/`|\bimport\b|\bexport\b|\byield\b/)) {
```

Comment:
So which regex would you recommend? ``/`|\b(?:import|export|yield)\b/``?

FWIW I think `import` is the most likely substring that a file would contain, and most likely to be first. Backticks are probably last.